app/controllers/groups_controller.rb

```diff
@@ -1,23 +1,29 @@
 class GroupsController < ApplicationController
-  before_action :set_group, only: %i[show edit update destroy]
-  load_and_authorize_resource
+  # before_action :set_group, only: %i[show edit update destroy]
+  # load_and_authorize_resource
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻